arb_os/inbox.mini

```diff
@@ -291,10 +311,7 @@ impure func translateUnsignedTx(request: IncomingRequest) -> option<TxRequestDat
         request = request with {
             requestId: uint(hash(
                 bytes32(request.sender),
-                hash(
-                    bytes32(chainParams_chainId()),
-                    marshalledBytes_hash(request.msgData)
-                )
+                hash(bytes32(chainParams_chainId()), marshalledBytes_hash(bytearray_marshalFull(request.msgData)))
```

Comment:
Potentially we should change what hash we use here to be more efficiently computable from a ByteArray